README.md

```diff
@@ -452,9 +452,14 @@ Other Style Guides
       if (subject === 'Mockingbird') {
         return author === 'Harper Lee';
       }
-
       return false;
     });
+    
+    //best
+     inbox.filter(({ subject, author } ) => {
+       return subject === 'Mockingbird' && author === 'Harper Lee';
+    });
+    
```

Comment:
this isn't actually best - our linter config will warn on it. it'd be:
```suggestion
    
    //best
     inbox.filter(({ subject, author }) => subject === 'Mockingbird' && author === 'Harper Lee');
```

however, then we've defeated the whole point of this section, which is showing patterns around explicit `return` in callbacks.

so, i think this is best reverted.